backend/tests/unit/components/fixtures/token.json

```diff
@@ -0,0 +1,66 @@
+{
+  "jti": "20c32131-4bca-4c65-95e8-2774861c0eda",
+  "exp": 1578010912,
+  "nbf": 0,
+  "iat": 1578010852,
+  "iss": "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/vehizw2t",
+  "aud": [
+    "realm-management",
+    "account"
+  ],
+  "sub": "4b5d0a2f-63a0-4c18-9d70-753648b8e7a9",
+  "typ": "Bearer",
+  "azp": "getok",
+  "nonce": "7308177a-712c-46ab-bdaa-aa2683aee8c7",
+  "auth_time": 1578010851,
+  "session_state": "bb26a935-c6ba-4f50-9261-2f4d7171c6d1",
+  "acr": "1",
+  "realm_access": {
+    "roles": [
+      "WORG",
+      "MSSC",
+      "WEBADE_CFG_READ",
+      "offline_access",
+      "uma_authorization",
+      "WEBADE_CFG_READ_ALL"
+    ]
+  },
+  "resource_access": {
+    "realm-management": {
+      "roles": [
+        "view-realm",
+        "view-identity-providers",
+        "manage-identity-providers",
+        "impersonation",
+        "realm-admin",
+        "create-client",
+        "manage-users",
+        "query-realms",
+        "view-authorization",
+        "query-clients",
+        "query-users",
+        "manage-events",
+        "manage-realm",
+        "view-events",
+        "view-users",
+        "view-clients",
+        "manage-authorization",
+        "manage-clients",
+        "query-groups"
```

Comment:
Just to be safe, let's not have all these "adminy" roles in here if we don't need it to test

Reply:
Removed the unneeded administrative roles